ucp/__init__.py

```diff
@@ -12,13 +12,13 @@
 from .public_api import *  # noqa
 from .utils import get_address  # noqa
 
+logger = logging.getLogger("ucx")
+
 # Notice, if we have to update environment variables
 # we need to do it before importing UCX
 if os.environ.get("UCX_MEMTYPE_CACHE", "") != "n":
     # See <https://github.com/openucx/ucx/wiki/NVIDIA-GPU-Support#known-issues>
-    warnings.warn(
-        "Setting env UCX_MEMTYPE_CACHE=n, which is required by UCX", UCXWarning
-    )
+    logger.debug("Setting env UCX_MEMTYPE_CACHE=n, which is required by UCX")
```

Comment:
I'm also curious about setting the default logging level below, and using the global logger.  @madsbk git blame points to you.  Can I ask you for your thoughts on the lines just below these?

Reply:
I haven't given it much thought :)
You are more then welcome to change it.